includes/EventManager.php

```diff
@@ -232,5 +250,10 @@ public function send_batch() {
 		$this->send( $events );
 
 		$queue->remove( $ids );
+
+		if ( $this->error['retryCount'] >= 1 ) {
+			\sleep( 15 );
```

Comment:
Hi @circlecube , I added a 15-second delay to push to the queue, can you let me know if it works?